### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by andymantell**

### CODE REVIEW

File: src/components/govukComponents/BackLink.js [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+    </Link>
+  )
 }
 
 BackLink.defaultProps = {
-  href: "/",
-  text: 'Back',
-  classes: ''
+  href: '/',
+  text: <>Back</>,
```
Comment (mick-jones) [inline src/components/govukComponents/BackLink.js]: Why not just a string?

File: src/components/govukComponents/BackLink.js [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+  classes: '',
+}
+
+BackLink.propTypes = {
+  attributes: PropTypes.object,
+  classes: PropTypes.string,
+  href: PropTypes.string,
+  html: PropTypes.node,
+  text: PropTypes.node,
```
Comment (mick-jones) [inline src/components/govukComponents/BackLink.js]: As above

File: src/components/govukComponents/CharacterCount.js [hunk 3]
```diff
@@ -34,14 +34,33 @@ function CharacterCount(props) {
         describedBy={characterCountInfoClass}
       />
       <span id={characterCountInfoClass} className="govuk-hint govuk-character-count__message" aria-live="polite">
-        You can enter up to {props.maxlength || props.maxwords} {props.maxwords ? 'words' : 'characters'}
+        You can enter up to
+        {' '}
```
Comment (mick-jones) [inline src/components/govukComponents/CharacterCount.js]: Left over from prettier?

File: src/components/govukComponents/CharacterCount.js [hunk 4]
```diff
@@ -34,14 +34,33 @@ function CharacterCount(props) {
...
+        {props.maxlength || props.maxwords}
+        {' '}
```
Comment (mick-jones) [inline src/components/govukComponents/CharacterCount.js]: As above

File: src/components/govukComponents/SummaryList.js [hunk 5]
```diff
@@ -1,62 +1,99 @@
...
-    </dd>
-  } else if(anyRowHasActions) {
+      </dd>
+    )
+  }
+
+  if (anyRowHasActions) {
     // Add dummy column to extend border
-    return <span className="govuk-summary-list__actions"></span>
+    return <span className="govuk-summary-list__actions" />
```
Comment (mick-jones) [inline src/components/govukComponents/SummaryList.js]: Self closing container tags?

File: src/components/govukComponents/Textarea.js [hunk 6]
```diff
@@ -1,38 +1,44 @@
...
   return (
-
     <div className={`govuk-form-group${props.errorMessage ? ' govuk-form-group--error' : ''} ${(props.formGroup && props.formGroup.classes) || ''}`}>
       <Label
         {...props.label}
         for={props.id}
       />
-      {hint}
-      {errorMessage}
+      { hint }
```
Comment (mick-jones) [inline src/components/govukComponents/Textarea.js]: shouldn't this be `{hint}`?

File: src/components/govukComponents/Textarea.js [hunk 7]
```diff
@@ -1,38 +1,44 @@
...
+      { errorMessage }
```
Comment (mick-jones) [inline src/components/govukComponents/Textarea.js]: As above

File: src/demos/PhaseBannerDemo.js [hunk 8]
```diff
@@ -1,16 +1,15 @@
...
-
   useEffect(() => {
     document.title = 'Phase banner demo'
   }, [])
 
   return (
     <main id="#content">
       <h1 className="govuk-heading-l">Phase banner</h1>
-      <PhaseBanner tag={{text: 'Beta'}} />
+      <PhaseBanner tag={{ text: 'Beta' }} />
```
Comment (mick-jones) [inline src/demos/PhaseBannerDemo.js]: remove padding

File: src/demos/SelectDemo.js [hunk 9]
```diff
@@ -36,7 +35,7 @@ function SelectDemo() {
         id="sort"
         name="sort"
         label={
-          { text: "Sort by" }
+          { text: 'Sort by' }
```
Comment (mick-jones) [inline src/demos/SelectDemo.js]: padding as above

File: src/components/govukComponents/BackLink.js (hunk 2 above)
Comment (andymantell) [reply src/components/govukComponents/BackLink.js]: The default value can be a string, and I have changed it, but PropTypes.node is there to catch all "renderable" things, not just strings. The contents could be jsx

File: src/components/govukComponents/SummaryList.js (hunk 5 above)
Comment (andymantell) [reply src/components/govukComponents/SummaryList.js]: Removed

**Pull request merged**